spec/models/episode_spec.rb

```diff
@@ -1,4 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe Episode, :type => :model do
+
+  let(:episode) {Epsisode.create(series_title: "Cheers",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.